roles/openshift_version/tasks/check_available_rpms.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Get available {{ openshift_service_type}} version
```

Comment:
I'm starting to wonder if it's a good idea to include variables in task names, seems to make it harder to locate offending tasks or am I mis-remembering that at certain log levels we don't get the line reference in the error?

Reply:
I'm not crazy about variable task names either.